CONTRIBUTING.md

```diff
@@ -15,11 +15,36 @@ to discuss proposals and use pull requests to suggest changes. For more
 information see the [Cilium Development
 Guide](https://docs.cilium.io/en/latest/contributing/development/).
 
+## An important note about mutating cluster resources
+
+Cilium CLI must follow these general rules regarding mutating cluster resources:
+
+- Cilium CLI must not mutate resources that are not managed by the Cilium Helm
```

Comment:
If we follow this policy, some of `cilium clustermesh` subcommand and `cilium multicast` subcommands, which I [just added](https://github.com/cilium/cilium-cli/pull/2620) is inappropriate?

Reply:
i consider `cilium multicast` to be one of the exceptional cases because there is no CRD to propagate the configuration to all the cilium agents.

i haven't looked at `cilium clustermesh` command for a while but at least clustermesh connection configuration is persisted as helm values: https://github.com/cilium/cilium-cli/blob/9577d6b14fc5ab28ffacc5e37e03df2086bc09a0/clustermesh/clustermesh.go#L1596

Reply:
I understand.
Thank you so much for kind description.

Reply:
> i consider `cilium multicast` to be one of the exceptional cases

Do you think this does not need to be written in CONTRIBUTING.md ?